### PR TITLE
Bump to 2.13.7 fixing incompats

### PR DESCRIPTION
**Pull request by som-snytt**

### CODE REVIEW

**Comment by som-snytt** [comment]
@SethTisue does that mean it worked?

File: build.sbt [hunk 1]
```diff
@@ -17,8 +17,8 @@ lazy val collectionContrib = crossProject(JVMPlatform, JSPlatform)
     name := "scala-collection-contrib",
     versionPolicyIntention := Compatibility.BinaryCompatible,
     scalaModuleAutomaticModuleName := Some("scala.collection.contrib"),
-    scalacOptions ++= Seq("-opt-warnings", "-language:higherKinds", "-deprecation", "-feature", "-Xfatal-warnings"),
-    Compile / doc / scalacOptions ++= Seq("-implicits", "-groups"),
+    Compile / compile / scalacOptions ++= Seq("-opt-warnings", "-language:higherKinds", "-deprecation", "-feature", "-Xfatal-warnings", "-Wconf:origin=scala.collection.IterableOps.toIterable:s"),
+    Compile / doc / scalacOptions ++= Seq("-implicits", "-groups", "-nowarn"),
```
Comment (som-snytt) [inline build.sbt:21]: leftover from endless attempts to silence warnings.

**Review by SethTisue — APPROVED** [review]

**Comment by som-snytt** [comment]
I guess it worked.

**Comment by SethTisue** [comment]
If a syntax tree fell in the forest and there was no CI to test it, did it issue a warning?

**Pull request merged**